src/Microsoft.Performance.Toolkit.Engine/EngineCreateInfo.cs

```diff
@@ -141,5 +60,23 @@ public EngineCreateInfo(IEnumerable<string> extensionDirectories)
         ///     The logger factory should be able to provide each processing source a logger specific to its type.
         /// </remarks>
         public Func<Type, ILogger> LoggerFactory { get; set; }
+
+        /// <summary>
+        ///     Gets the data sources that are to be processed by an <see cref="Engine"/>
+        ///     instance.
+        /// </summary>
+        public DataSourceSet DataSources { get; }
+
+        /// <summary>
+        ///     Gets or sets a value indicating whether the created <see cref="Engine"/>
+        ///     instance owns the data sources passed in <see cref="DataSources"/>.
```

Comment:
I would either say "passed in (to the ctor)" or just "owns <see cref="DataSources"/>